### PR TITLE
refactor translations.js

### PR DESCRIPTION
**Pull request by peterbe**
(based on an offline (Slack) conversation)

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ async function buildDocumentFromURL(url) {
     // When you're running the dev server and build documents
     // every time a URL is requested, you won't have had the chance to do
     // the phase that happens when you do a regular `yarn build`.
-    document.translations = findDocumentTranslations(document);
+    document.translations = findDocumentTranslationsByURL(document);
```
Comment (fiji-flo) [inline server/index.js:50]: nit: I think `findDocumentTranslationsByURL` accepting a `document` is a little bit misleading maybe we wanna pass just `document.url`

**Review by fiji-flo — APPROVED** [review]
Feel free to ignore/address my nit

File: server/index.js (hunk 1 above)
Comment (peterbe) [reply server/index.js]: You're right. I'm glad you stopped me. Fixed now.

**Comment by peterbe** [comment]
Oh I see. You approved but + nit. I'll merge.

**Pull request merged**